addon/utils/uuid.js

```diff
@@ -1 +1,17 @@
-export { _utilsUuid as default } from 'miragejs';
+import { _utilsUuid } from 'miragejs';
+import { deprecateNestedImport } from '../deprecate-imports';
+
+/**
+ @function uuid
+ @hide
+ */
+export default function uuid(...args) {
+  const message =
+    `Importing 'uuid' from 'ember-cli-mirage/utils/reference-sort' is deprecated. ` +
```

Comment:
I assume this was meant to be `ember-cli-mirage/utils/uuid`?

Reply:
@Turbo87 indeed! created #2306 to fix this

Reply:
@Turbo87 Released in [v2.3.1](https://github.com/miragejs/ember-cli-mirage/releases/tag/v2.3.1)